doc/cygnus-ngsi/flume_extensions_catalogue/ngsi_mongo_sink.md

```diff
@@ -286,6 +287,25 @@ If `data_model=dm-by-entity` and `attr_persistence=column` then `NGSIMongoSink`
 
 [Top](#top)
 
+### <a name="section1.4"></a>MongoDB Indexes
+
+Since version 3.0.0 of cygnus index named `cyg_agg_opt` is created in this way but depending on datamodel:
```

Comment:
Should both indexed (cyg_agg_opt and cyg_raw_opt) be mentioned in both sinks (ngsi_mongo_sink.md and ngsi_sth_sink.md)?

As far as I remember (although I may be wrong) each index is used by one sink but I don't remember if it is this way

* mongo sink -> cyg_agg_opt
* sth sink -> cyg_raw_opt

or

* mongo sink -> cyg_raw_opt
* sth sink -> cyg_agg_opt

Reply:
> Should both indexed (cyg_agg_opt and cyg_raw_opt) be mentioned in both sinks (ngsi_mongo_sink.md and ngsi_sth_sink.md)?
> 
> As far as I remember (although I may be wrong) each index is used by one sink but I don't remember if it is this way
> 
>     * mongo sink -> cyg_agg_opt
> 
>     * sth sink -> cyg_raw_opt
> 
> 
> or
> 
>     * mongo sink -> cyg_raw_opt
> 
>     * sth sink -> cyg_agg_opt

@fgalan you are rigth!. MongoSink creates Raw table and STHSInk creates Aggregate table.